apps/etterlatte-saksbehandling-ui/client/src/components/person/Person.tsx

```diff
@@ -112,7 +112,7 @@ export const Person = () => {
                 icon={<PersonIcon />}
               />
               <Tabs.Tab value={PersonOversiktFane.HENDELSER} label="Hendelser" icon={<BellIcon />} />
-              {isOmstillingsstoenad(sakResult) && skalViseAktivitet && (
+              {isOmstillingsstoenad(sakResult) && !skalViseAktivitet && (
```

Comment:
```suggestion
              {isOmstillingsstoenad(sakResult) && skalViseAktivitet && (
```
Sikkert blitt med fra testing :D